SalesforceSDK/Analytics/Store/EventStoreManager.cs

```diff
@@ -162,14 +159,6 @@ public EventStoreManager(string fileNameSuffix, string encryptionKey)
             }
         }
 
-        public async Task ChangeEncryptionKeyAsync(string oldKey, string newKey)
```

Comment:
The passcode changes/encryption key changes are handles by the Encryptor class that is implemented in the Universal project.
https://github.com/forcedotcom/SalesforceMobileSDK-Windows/blob/unstable/SalesforceSDK/Universal/Security/Encryptor.cs